01-js-syntax/exercise/02-greatest-common-divisor.js

```diff
@@ -0,0 +1,15 @@
+function gdc(first, second) {
+
+    if (second === 0) {
```

Comment:
Why not deleting this empty file and trying to make the function shorter with ternary operator?
IMO it will look better.